stopcovid/clients/sqs.py

```diff
@@ -20,9 +14,9 @@ def publish_outbound_sms_messages(outbound_sms_messages: List[OutboundSMS]):
 
     entries = [
         {
-            "Id": outbound_sms.event_id,
+            "Id": str(outbound_sms.event_id),
```

Comment:
these are UUIDs? That's why you're converting?

Reply:
Yeah -- we should probably be casting these to strings higher up. Thoughts?

Reply:
this might actually be failing due to a test case. if so, i should be stringifying there. Let me look into it